### ISSUE TITLE
Changes marked as `deleted` for updated cases with updated watch library

### ISSUE THREAD

**Issue by robotpony** [lemonstand/lemonsync-js#32]
Updates to use `node-watch` are causing odd behaviour on Mac OS (needs more testing).

```bash
🍋  Watching for changes... 🍋

- /Users/mx/ls-dev/themes/lscloud-theme-bones/partials/product.htm deleted
```

**Comment by robotpony** [comment]
Fixed in #33

**Comment by robotpony** [comment]
Fixed in PR #33
